tests/auto_inject/test_auto_inject_install.py

```diff
@@ -75,6 +74,15 @@ def test_profiling(self, virtual_machine):
         self._test_install(virtual_machine, profile=True)
 
 
+@features.crashtracking
+@scenarios.container_auto_injection_install_script_crashtracking
+class TestContainerAutoInjectInstallScriptCrashTracking(base.AutoInjectBaseTest):
+    @parametrize_virtual_machines()
+    @missing_feature(reason="Crash logs not currently forwarded to org 749096")
```

Comment:
But we are testing docker ssi, this test class if for the aws ssi. It doesn't make sense in this PR